packages/renderer/src/lib/image/PullImage.svelte

```diff
@@ -268,7 +288,9 @@ function checkIfTagExist(image: string, tags: string[]): void {
           id="imageName"
           name="imageName"
           placeholder="Image name"
-          searchFunction={searchImages}
+          onInputChange={searchFunction}
+          resultItems={searchResult}
+          sorted
```

Comment:
This can be confusing. We are generally asking the component to do things (hidden -> please hide the content; disabled -> please disable the component), here we are saying not to do it (sorted -> the content is already sorted, please don't do anything)